tsconfig.json

```diff
@@ -1,5 +1,31 @@
 {
-  "extends": "./tsconfig.base.json",
-  "include": ["test/**/*.test.ts", "test/**/*.test.tsx"],
-  "exclude": ["node_modules"]
+  "compilerOptions": {
+    "strict": false,
+    "noEmit": true,
+    "allowJs": true,
+    "resolveJsonModule": true,
+    "jsx": "react-jsx",
+    "module": "esnext",
+    "target": "ESNext",
+    "esModuleInterop": true,
+    "moduleResolution": "node",
+    "types": ["react", "jest", "node", "trusted-types", "jest-extended"],
+    "paths": {
+      "development-sandbox": ["./test/lib/development-sandbox"],
+      "next-test-utils": ["./test/lib/next-test-utils"],
+      "amp-test-utils": ["./test/lib/amp-test-utils"],
+      "next-webdriver": ["./test/lib/next-webdriver"],
+      "e2e-utils": ["./test/lib/e2e-utils"],
+      "test-data-service/*": ["./test/lib/test-data-service/*"],
+      "test-log": ["./test/lib/test-log"]
+    }
+  },
+  "include": ["test/**/*.ts", "test/**/*.test.tsx", "test/lib/**/*.ts"],
+  "exclude": [
+    "test/development/acceptance-app/fixtures/app-hmr-changes/mdx-components.ts",
+    "test/e2e/opentelemetry/instrumentation",
+    "test/e2e/opentelemetry/client-trace-metadata/instrumentation.ts",
+    "test/integration/typescript-paths",
+    "test/integration/typescript-workspaces-paths"
```

Comment:
Excluding these is unfortunate, but they have their own tsconfigs and/or type dependencies (e.g. `@opentelemetry/...`) and therefore can't really be compiled from the root, since we're not using project references here.

Reply:
Do these extend the root tsconfig to get access to typings of our test utils?

Reply:
I've now removed the excludes. This was only an issue because I accidentally changed the first include pattern from `test/**/*.test.ts` to `test/**/*.ts`, which I've reverted now.